website/content/docs/job-specification/ui.mdx

```diff
@@ -0,0 +1,47 @@
+---
+layout: docs
+page_title: ui Block - Job Specification
+description: |-
+  The "ui" block allows lets users add description and helpful links to their
+  job page in the Nomad Web UI.
+---
+
+# `ui` Block
+
+<Placement
+  groups={[
+    ['job', 'ui'],
+  ]}
+/>
+
+The `ui` block provides options to modify the presentation of the Job index page
+in the Web UI. When specified, a description and any number of links will be
+added to the top of the job page in question.
+
+The following will provide the Web UI with a job description and a pair of links:
+
+## `ui` Parameters
+
+- `description` `(string: "")` - The markdown-enabled description of the job
```

Comment:
We're calling this "markdown-enabled" but we should probably tell the user what markdown dialect is permitted. Ex. is this GitHub-flavored markdown, commonmark, etc.? Also, is there any markdown feature that's disallowed here (ex. can the user add html markup and XSS the cluster admin, or is that disabled?)

Reply:
Thanks for this comment; I've added render sanitization via https://github.com/hashicorp/nomad/pull/18292/commits/ed9badf7de6b9443b0d178aaeb2356bb6bb49491